docs/getting_started.rst

```diff
@@ -85,14 +85,14 @@ URLs defined by that backend. You can do the same with the
 Auto slug field
 ---------------
 
-Historically Django-Organizations relied on `django-extensions
+Django-Organizations relies on `django-extensions
 <http://django-extensions.readthedocs.org/en/latest/>`_ for the base
 `AutoSlugField
 <http://django-extensions.readthedocs.org/en/latest/field_extensions.html>`_.
 While django-extensions is great, this does require that every project install
 django-extensions for this one small feature.
 
-If you decide to use the default django-organization models by adding
+If you choose not to use the default django-organization models by adding
```

Comment:
I'm not sure that wording change is accurate -- I think the original wording may still be true. But my brain is tired. Can you confirm one way or the other, @bennylope?

Reply:
yes I think so too @AZtheAsian I think is better you cancel that change